<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="KH-HYD-35" name="com.kh.IRA.skillmap.AddandEdit_skill" tests="5" failures="0" timestamp="2021-06-01T21:03:36 IST" time="6.442" errors="1">
  <testcase name="addSkillSubskillCompetency" time="2.582" classname="com.kh.IRA.skillmap.AddandEdit_skill"/>
  <system-out/>
  <testcase name="editSkillSubskillCompetency" time="0.549" classname="com.kh.IRA.skillmap.AddandEdit_skill">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 2, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 2, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:657)
at java.util.ArrayList.get(ArrayList.java:433)
at com.kh.IRA.pagehelper.SkillMapPageHelper.edit(SkillMapPageHelper.java:74)
at com.kh.IRA.skillmap.AddandEdit_skill.editSkillSubskillCompetency(AddandEdit_skill.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- editSkillSubskillCompetency -->
  <system-out/>
  <testcase name="editSkillSubskillCompetency" time="1.398" classname="com.kh.IRA.skillmap.AddandEdit_skill">
    <skipped/>
  </testcase> <!-- editSkillSubskillCompetency -->
  <system-out/>
  <testcase name="editSkillSubskillCompetency" time="0.442" classname="com.kh.IRA.skillmap.AddandEdit_skill">
    <skipped/>
  </testcase> <!-- editSkillSubskillCompetency -->
  <system-out/>
  <testcase name="editSkillSubskillCompetency" time="1.471" classname="com.kh.IRA.skillmap.AddandEdit_skill">
    <skipped/>
  </testcase> <!-- editSkillSubskillCompetency -->
  <system-out/>
</testsuite> <!-- com.kh.IRA.skillmap.AddandEdit_skill -->
